packages/venia-concept/src/components/Checkout/form.js

```diff
@@ -196,9 +192,22 @@ class Form extends Component {
             return <span className={promptClass}>Add Billing Information</span>;
         }
 
+        let primaryDisplay = '';
+        let secondaryDisplay = '';
+        if (paymentData) {
+            primaryDisplay = paymentData.details.cardType;
```

Comment:
Is `details` always an object? What is `cardType` for check/MO?

Reply:
For right now it is - check/MO has been completely removed from the application.

If/when other payment methods get added this will definitely have to be updated to handle them.